README.md

```diff
@@ -32,6 +32,8 @@ Pelita is a PacMan™ like game. Two teams each of two bots are placed in a maze
 
 - **game over**: the game ends when one team eats all of its enemy's food pellets or after **300** rounds.
 
+- **losing**: if an error is raised by a team during the game, the team will disconnect and it will be an automatic loss.  
```

Comment:
Maybe we could call it `Exception` instead of `error`, so that it is clear what is meant by it. Then, I'd say that the team _will lose the game regardless of the current score_, or something like that (being _disconnected_ is referring to an internal implementation detail, no? We never speak about _connecting_ in the docs either). What do you think?

Reply:
Valid point but the UI will print `Disqualified: disconnected` so maybe this gives the students a hint what has happened.

Reply:
OK, then fix the UI? We don't speak about connections at all in the client docs...

Reply:
The Pelita code won’t be touched until after ASPP-Apac-2019 so that’s not going to help them anyway. :)